src/anonfile/anonfile.py

```diff
@@ -146,9 +146,18 @@ def name(self) -> Path:
     @property
     def size(self) -> int:
         """
-        Return the uploaded file size in bytes.
+        Return the uploaded file size in bytes as an integer.
         """
         return int(self.json['data']['file']['metadata']['size']['bytes'])
+    
+    # It was the only dedicated response missing from the original json query
```

Comment:
You should remove this comment, put it in a commit message if you want to but don't add these remarks in the code directly.